README.md

```diff
@@ -114,6 +114,7 @@ interface RequestBasketItemData {
     faceValueInShopperCurrency?: Amount | null;
     adjustmentAmountInShopperCurrency?: Amount;
     salePriceInShopperCurrency?: Amount;
+    feeInShopperCurrency?: Amount | null;
```

Comment:
Is there nowhere on README.md that needed location added?

Reply:
No I don't believe so